WordPress/src/main/java/org/wordpress/android/ui/plugins/PluginListFragment.java

```diff
@@ -36,8 +37,14 @@
 
 import java.util.List;
 
+import javax.inject.Inject;
+
 public class PluginListFragment extends Fragment {
     public static final String TAG = PluginListFragment.class.getName();
+
+    @Inject
+    ViewModelProvider.Factory mViewModelFactory;
```

Comment:
Similar to my comment before, we keep the `@Inject`s in the same line as the property.